src/server/connection.mjs

```diff
@@ -190,6 +191,9 @@ export class ServerConnection {
     else if (packet.type == ClosePayload.type) {
       await this.close_stream(packet.stream_id, packet.reason);
     }
+    else if (packet.type == InfoPayload.type) {
+      // TODO: Handle this shit.
```

Comment:
This whole PR is just unfinished with questionable code quality so I'm not merging it.